nixos/modules/services/autotierfs.nix

```
@@ -0,0 +1,87 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.autotierfs;
+  ini = pkgs.formats.ini { };
+  format = lib.types.attrsOf ini.type;
+
+  configName = builtins.replaceStrings [ "/" ] [ "_" ];
```

Maybe something like this? Otherwise, the name gives the impression that it's a value rather than a function

```suggestion
  generateConfigName = builtins.replaceStrings [ "/" ] [ "_" ];
```